index.html

```diff
@@ -1,49 +1,189 @@
+                    <li><img src="images\Counter.svg"></li>
+                    <li class="item">Back End Dev</li>
+                    <li><img src="images\Counter.svg"></li>
+                    <li class="item">2022</li>
+                </ul>
+            </section>
+            <p class="project-description">
+                A daily selection of privately personalized 
+                reads; no accounts or sign-ups required.
+            </p>
+            <ul class="coding-languages">
+                <li class="item-languages">html</li>
+                <li class="item-languages">css</li>
+                <li class="item-languages">javaScript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+        </section>
+        <section class="tonic-2">
+            <img class="photo-grid" src="images\tonic2.svg" alt="Project picture">
+            <h1 class="project-title">Tonic</h1>
+            <section class="canopy-segment">
+                <h2 class="canopy">CANOPY</h2>
+                <ul class="canopy-items">
+                    <li><img src="images\Counter.svg"></li>
+                    <li class="item">Back End Dev</li>
+                    <li><img src="images\Counter.svg"></li>
+                    <li class="item">2022</li>
+                </ul>
+            </section>
+            <p class="project-description">
+                A daily selection of privately personalized 
+                reads; no accounts or sign-ups required.
+            </p>
+            <ul class="coding-languages">
+                <li class="item-languages">html</li>
+                <li class="item-languages">css</li>
+                <li class="item-languages">javaScript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+        </section>
+        <section class="stories-2">
+            <img class="photo-grid" src="images\stories2.svg" alt="Project picture">
+            <h1 class="project-title">Multi-Post Stories</h1>
+            <section class="canopy-segment">
+                <h2 class="canopy">CANOPY</h2>
+                <ul class="canopy-items">
+                    <li><img src="images\Counter.svg"></li>
+                    <li class="item">Back End Dev</li>
+                    <li><img src="images\Counter.svg"></li>
+                    <li class="item">2022</li>
+                </ul>
+            </section>
+            <p class="project-description">
+                A daily selection of privately personalized 
+                reads; no accounts or sign-ups required.
+            </p>
+            <ul class="coding-languages">
+                <li class="item-languages">html</li>
+                <li class="item-languages">css</li>
+                <li class="item-languages">javaScript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+        </section>
+
+    </div>
+    <div class="about-section">

```

Comment:
- The about should use a section tag and not a div here because this is another section in your portfolio, using the section tag here will be semantically correct. kindly fix this issue.